data/map planets.txt

```diff
@@ -5484,6 +5506,13 @@ planet "Spera Anatrusk"
 	landscape land/badlands11
 	description `This is a relatively dry planet, and the only Korath cities were close to the poles where the temperatures are more mild and the lifeless deserts give way to grasslands and even a few scattered forests. The largest city was clearly destroyed by orbital bombardment. The other cities seem to have been done in by rioting or looting, leaving smashed windows and burned-out buildings behind. Some of the damage is very recent; the Korath survivors here held out for a long time.`
 
+planet "Spiraling Indigo"
+	attributes "gas giant" "requires: gaslining" uninhabited
+	landscape land/nasa38
+	description `Spiraling Indigo is the heaviest gas giant in Coalition space, exceeded only by stars and ringworlds. While the Kimek certainly didn't shy away from harvesting the plentiful resources to be found here, the planet has had much more of a cultural footprint than an economic one. The perpetual storms surrounding the poles here have inspired many Kimek artists, be they writers, stylists, musicians, painters, or of any other kind. The world's popularity in Kimek media has even prompted some religious institutions to claim its beauty as a divine signal.`
```

Comment:
I think it might be better to remove the last part of the sentence. The list of kinds of artists doesn't need to be exhaustive, and I think it sounds better if the sentence is more concise.

```suggestion
	description `Spiraling Indigo is the heaviest gas giant in Coalition space, exceeded only by stars and ringworlds. While the Kimek certainly didn't shy away from harvesting the plentiful resources to be found here, the planet has had much more of a cultural footprint than an economic one. The perpetual storms surrounding the poles here have inspired many Kimek artists, be they writers, stylists, musicians, or painters. The world's popularity in Kimek media has even prompted some religious institutions to claim its beauty as a divine signal.`
```